packages/devtools_app/lib/src/shared/ui/filter.dart

```diff
@@ -130,12 +130,12 @@ mixin FilterControllerMixin<T> on DisposableController
   }
 }
 
-/// Dialog to manage filter settings.
+/// Dialog to manage toggleable filter settings.
 ///
 /// This dialog interacts with a [FilterControllerMixin] to manage and preserve
-/// the filter state managed by the dialog.
-class FilterDialog<T> extends StatefulWidget {
-  FilterDialog({
+/// the toggleable filter state managed by the dialog.
+class ToggleFilterDialog<T> extends StatefulWidget {
```

Comment:
We should undo this rename since this isn't only for toggle filters. This dialog also contains a query filter by default.

Reply:
This was accidental, It is flipped back now :)